honeybadgermpc/equality.py

```diff
@@ -0,0 +1,107 @@
+from honeybadgermpc.mpc import PreProcessedElements, Subgroup, TaskProgramRunner
+import logging
+import asyncio
+from honeybadgermpc.field import GF
+from honeybadgermpc.mixins import BeaverTriple, MixinOpName
+SECURITY_PARAMETER = 32
```

Comment:
This is a parameter that can passed as an argument to the equality function. It would also make sense to define such parameters in a config file, since we will often set `STATISTICAL_SECURITY_PARAMETER` separately from the computational security parameters